app.bluebubbles.BlueBubbles.yml

```diff
@@ -0,0 +1,189 @@
+app-id: app.bluebubbles.BlueBubbles
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: bluebubbles
+separate-locales: false
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --own-name=app.bluebubbles.BlueBubbles
+  - --system-talk-name=org.freedesktop.DBus
+  - --system-talk-name=org.freedesktop.DBus.*
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --system-talk-name=org.freedesktop.NetworkManager.Connection.Active
+  - --system-talk-name=org.freedesktop.FileManager1
+  - --share=network
+  - --filesystem=xdg-data
+  - --filesystem=xdg-download
+modules:
+  - name: bluebubbles
+    buildsystem: simple
+    build-commands:
+      - 'mkdir -p bluebubbles'
+      - 'tar -xf bluebubbles-linux-portable.tar.gz -C bluebubbles'
+      - 'cp -r bluebubbles /app/'
+      - 'chmod +x /app/bluebubbles/bluebubbles'
+      - 'mkdir -p /app/bin'
+      - 'ln -s /app/bluebubbles/bluebubbles /app/bin/bluebubbles'
+      - 'mkdir -p /app/lib'
+      - 'mv libvlc.so.5/lib64/* /app/lib/'
+      - 'mv libappindicator3.so.1/lib64/* /app/lib/'
+      - 'mv libindicator3.so.7/lib64/* /app/lib/'
+      - 'mv libdbusmenu-glib.so.4/lib64/* /app/lib/'
+      - 'mv libvlccore.so.9/lib64/* /app/lib/'
+      - 'mv libdbusmenu-gtk3.so.4/lib64/* /app/lib/'
+      - 'mv libidn.so.11/lib64/* /app/lib/'
+      - 'mv libobjectbox.so/libobjectbox.so /app/lib/'
+      - 'cp zenity/bin/zenity /app/bin/'
+      - 'mkdir -p /app/share/metainfo'
+      - 'cp -r app.bluebubbles.BlueBubbles.metainfo.xml /app/share/metainfo/'
+      - 'mkdir -p /app/share/icons/hicolor/64x64/apps'
+      - 'cp -r 64x64.png /app/share/icons/hicolor/64x64/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/icons/hicolor/128x128/apps'
+      - 'cp -r 128x128.png /app/share/icons/hicolor/128x128/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/icons/hicolor/256x256/apps'
+      - 'cp -r 256x256.png /app/share/icons/hicolor/256x256/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/icons/hicolor/512x512/apps'
+      - 'cp -r 512x512.png /app/share/icons/hicolor/512x512/apps/app.bluebubbles.BlueBubbles.png'
+      - 'mkdir -p /app/share/applications'
+      - 'cp -r app.bluebubbles.BlueBubbles.desktop /app/share/applications/'
+      - 'mkdir -p /app/share/appdata'
+      - 'cp -r app.bluebubbles.BlueBubbles.metainfo.xml /app/share/appdata/'
+    sources:
+      - type: file
+        path: bluebubbles-linux-portable.tar.gz
```

Comment:
this should NOT be in this repository. This should be downloaded from upstream. Even more  since this is a 56MB archive.